src/parquet/column/column-reader-test.cc

```diff
@@ -128,7 +129,8 @@ TEST_F(TestPrimitiveReader, TestInt32FlatRequired) {
   max_def_level_ = 0;
   max_rep_level_ = 0;
   NodePtr type = schema::Int32("a", Repetition::REQUIRED);
-  const ColumnDescriptor descr(type, max_def_level_, max_rep_level_);
+  const ColumnDescriptor descr(type, max_def_level_, max_rep_level_,
+      ColumnPath::FromDotString("a"));
```

Comment:
Now that we have the `parent` pointers, we don't need to construct the column path at construction time, but we can always build it by following the parent links.